### ISSUE TITLE
More Support

### ISSUE THREAD

**Issue by bradleypeabody** [gotailwindcss/tailwind#1]
See what it would take to implement all of this:

https://tailwindcss.com/docs/functions-and-directives

**Comment by allain** [comment]
I'm amazed that this exists and would love to see this pushed on.

With the current wave of slow node.js tools being ported to golang (and rust), I think the iron's hot.

**Comment by bradleypeabody** [comment]
Yeah, I haven't had a chance to work on this but it's an interesting idea and so far is quite functional for the relatively small amount of work put into it.  (And yes, it's fast :) )
